### ISSUE TITLE
Option to disable log

### ISSUE THREAD

**Issue by cronfy** [XOP/postcss-rgba-hex#1]
While replacing colors, postcss-rgba-hex posts info messages to console:

```
RGB(a) replaced: rgb(224, 4, 59) -> #e0043b
RGB(a) replaced: rgb(251, 79, 123) -> #fb4f7b
RGB(a) replaced: rgb(250, 54, 104) -> #fa3668
RGB(a) replaced: rgb(77, 77, 77) -> #4d4d4d
```

Would be great to have an option to disable these messages.

**Comment by XOP** [comment]
@cronfy thanks, will take a peek!

**Comment by XOP** [comment]
addressed in v0.3.7  
https://github.com/XOP/postcss-rgba-hex/commit/c3d63abdecdf49785a362965db7f7f56f425045f  
https://www.npmjs.com/package/postcss-rgba-hex

**Comment by cronfy** [comment]
Just works! Thank you!
